particle-iot/particle-cli
# chore/dependencies

## Description

moved `serialport` to `optionalDependencies`, updated dependencies including `particle-usb`, updated install / update instructions to minimize confusion for users of our wrapper bins.


## How to Test

1. Clone this repository `$ git clone git@github.com:particle-iot/particle-cli.git && cd ./particle-cli`
2. Checkout this branch `$ git checkout chore/dependencies`
3. Install dependencies `$ npm install`
4. Connect a device
5. run `$ npm start -- usb start-listening`
6. run `$ npm start -- serial inspect`
7. run `$ npm start -- usb stop-listening` 

your device should enter listening mode, provide info about its current configuration, then exit listening mode.
